docs/configfile.adoc

```diff
@@ -21,6 +21,11 @@ copyToClipboard:config-file[]
 <8> This sets last commit sha or `HEAD` notation for inspecting changes.
 <9> This defines if smart testing should auto correct misspelled strategies.
 <10> This defines the pair key/value of custom strategies as list.
+<11> This defines list of strategies with it's configuration that you want to apply while applying strategy.
```

Comment:
> list of strategies

list of strategies is in `strategies` field. Here is a list of strategy related configurations